src/skimage/feature/blob.py

```diff
@@ -276,6 +296,29 @@ def blob_dog(
         `exclude_border`-pixels of the border of the image.
         If zero or False, peaks are identified regardless of their
         distance from the border.
+    prescale : {'minmax', 'legacy', False} or tuple[float, float], optional
```

Comment:
Maybe instead of copying this docstring everywhere, refer to a central docstring? I.e., we could expose the utility function in the public namespace? Or maybe this becomes templated, and is filled out on import?

Reply:
My personal pet peeve is docstring that are only useful at runtime. E.g., I find it really annoying when digging through SciPy's source code. 

I'm also -1 on hiding the information from users behind a link for the same reason.

If you are worried about this getting out of sync. I'd rather have a decorator enforcing the consistency at runtime.

Reply:
I agree that Scipy's automated docstrings are pretty annoying - and I think I wrote some of them.    I quite like the idea of enforcing the match between docstrings with a decorator.   We could add a source processor to do it, as well, to be run if the decorator check fails.

Reply:
That sound's like the best option. For now, I'll just keep the docstring in one place and leave the others empty.